feat: Added tutorial on 'npm Enterprise with Nexus repository'.

ritsingh opened the pull request:
Details: Added steps to be followed while configuring npm Enterprise with Nexus repository.

Commit:
feat: Added tutorial on 'npm Enterprise with Nexus repository'.

arobson commented on tutorials/npm-enterprise-with-nexus.md:
```diff
@@ -0,0 +1,45 @@
+
+
+* Open *http://your-host:8081/* in your browser.
+* Sign in with admin credentials ( You can crate your own user and login with that credentials)
```
Minor corrections:
 * Typo: crate -> create
 * that credentials -> those credentials

arobson commented on tutorials/npm-enterprise-with-nexus.md:
```diff
@@ -0,0 +1,45 @@
+    E.g *http://your_npme-registry-url/*
+3. Save the changes.
+
+Here’s a good documentation that explains better about configuring Nexus as npm repository *http://blog.sonatype.com/using-nexus-3-as-your-repository-part-2-npm-packages*
```
Minor correction: "This blog post provides more detail about configuring Nexus as an npm repository: "

arobson commented on tutorials/npm-enterprise-with-nexus.md:
```diff
@@ -0,0 +1,45 @@
+1. Select *Security->Realms* from the left panel options.
+2. Add `npm Bearer Token Realm` from available realms to active realms and save the changes.
+
+Here’s the documentation regarding authentication for npm nexus repository *http://www.sonatype.org/nexus/2015/11/19/new-npm-tools-for-your-nexus-repository-manager-tool-box/*.
```
Minor change: "This blog post provides more detail about authentication in Nexus for npm repositories: "

Commit:
feat: Updated docs for 'npm Enterprise with Nexus repository'.

Details:
Updated the suggested changes for nexus repository setup and authentication.

arobson reviewed (approved):
Thanks for making the updates 👍
